src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -125,4 +125,86 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners {
     runTester(new InternalBoreTester, annotations = TesterDriver.verilatorOnly) should be(true)
   }
 
+  it should "work using new API" in {
```

Comment:
havent read the test in detail, but can you show boring up into a parent and back into a sibling...? or how would that use case be handled?

Reply:
The later test shows this is illegal - for right now I'm only letting a user bore downwards.

Reply:
Do I need to support that?

Reply:
It would be good to allow it to enable the existing Grand Central View API.

This has taps that reach into the parent. Anything in the parent's visible `Data` can be captured. An example use is approximately:

```scala
class Bar extends RawModule {
  val a = Wire(Bool())
}

class Foo extends RawModule {
  val bar = Module(new Bar)
  View {
    val b = Tap(bar.a)
  }
}
```

The result of this is something like the following FIRRTL:

```
module Bar:
  output a_probe: Probe<UInt<1>>
  wire a: UInt<1>

  define(a_probe, probe(a))

module View_companion:
  input a: UInt<1>

  node b = a

module Foo:

  inst bar of Bar
  inst view of View_companion

  view.a <= read(bar.a_probe)
```

Reply:
right, we don't need a "secret IO" for that use case since the modules in question (the View and the Parent) are still open, but the Bore API should support at least one level of up-boring (TBD whether the bore API is supposed to work arbitrarily for some sort of 
"i promise this is accessible from some heirarchy if you keep going up both mine and the boring-top-point you'll find something in common")